[RHDHBUGS-2722]: Restructure authentication title to JTBD outline (#2049)
redhat-developer/red-hat-developers-documentation-rhdh: modules/shared/proc-share-a-secret-with-gitlab.adoc

:_mod-docs-content-type: PROCEDURE

[id="share-a-secret-with-gitlab_{context}"]
= Share a secret with GitLab

[role="_abstract"]
Register a GitLab OAuth 2 application and store the credentials in {product} to enable secure communication.

.Prerequisites
include::snip-enable-user-authentication-with-gitlab-common-prerequisites.adoc[]

.Procedure

> . Register GitLab OAuth 2 application to allow {product-very-short} to authenticate with GitLab. You must use the required callback URL and permissions.

Reviewer: ```suggestion
. Register the GitLab OAuth 2 application.
+
[IMPORTANT]
====
You must use the required callback URL and permissions.
====
+
```
